packages/react/src/FilteredActionList/FilteredActionListWithDeprecatedActionList.tsx

```diff
@@ -40,16 +45,19 @@ const StyledHeader = styled.div`
 
 export function FilteredActionList({
   loading = false,
+  loadingType = FilteredActionListLoadingTypes.bodySpinner,
   placeholderText,
   filterValue: externalFilterValue,
   onFilterChange,
+  onListContainerRefChanged,
+  onInputRefChanged,
   items,
   textInputProps,
-  inputRef: providedInputRef,
   sx,
   className,
   ...listProps
 }: FilteredActionListProps): JSX.Element {
+  const inputRef = useRef<HTMLInputElement>(null)
```

Comment:
does this mean consumers will no longer be able to provide their own refs?

Reply:
Oh man yeah, good catch. I meant to go back and clean this up!